Makefile

```
@@ -172,4 +172,48 @@ docforge-ci: docforge-download ## Run docforge in CI mode (non-interactive)
+			curl -sL https://github.com/vale-cli/vale/releases/download/v3.14.2/vale_3.14.2_Windows_64-bit.zip -o bin/vale.zip; \
+			unzip -q bin/vale.zip vale.exe -d bin; \
+			rm -f bin/vale.zip; \
+		else \
+			echo "Unsupported OS. Install Vale manually: https://vale.sh/docs/install"; \
+			exit 1; \
+		fi; \
+		echo "Vale installed to bin/. Add bin/ to your PATH or run: bin/vale"; \
+	fi
+
+.PHONY: vale-run
+vale-run: ## Lint changed website markdown files with Vale
```

checkmake (0.3.2)

[warning] Target body for "vale-run" exceeds allowed length of 5 lines (8).

(maxbodylength)